src/main/kotlin/com/exactpro/th2/lwdataprovider/SseEvent.kt

```diff
@@ -82,6 +85,15 @@ sealed class SseEvent(
 
     companion object {
 
+        @OptIn(ExperimentalSerializationApi::class)
+        private val JSON: ThreadLocal<Json> = object : ThreadLocal<Json>() {
+            override fun initialValue(): Json {
+                return Json {
+                    explicitNulls = false
+                }
+            }
+        }
```

Comment:
Maybe I am missing something but the Json class looks thread-safe to me.
There was an issue regarding that but it is fixed
https://github.com/Kotlin/kotlinx.serialization/issues/1455

If there are other considerations why do we need thread-local here let's leave them in the comment to the property